### ISSUE TITLE
About complete code for evaluation

### ISSUE THREAD

**Issue by tokaka22** [ForestsKing/ChatTime#8]
Thank you for your outstanding work!
 I have roughly used your repository and noticed that it doesn't seem to provide the complete code for evaluation. The demo only includes a single data sample. Or did I perhaps miss this part? Could you kindly share the code for this section as a reference?

**Comment by tokaka22** [comment]
I have tried using the [Time-Series-Library](https://github.com/thuml/Time-Series-Library/tree/main/data_provider)'s data loading parts and directly loaded the [ChengsenWang/ChatTime-1-7B-Chat](https://huggingface.co/ChengsenWang/ChatTime-1-7B-Chat) weights you provided, but the results I reproduced don't match the ones in the paper.

**Comment by JoffreyMa** [comment]
@ForestsKing Thank you for your work indeed and for open-sourcing it.
I am also interested in the code for evaluation, and in the test data, which are not in the repository nor HuggingFace.

@tokaka22 have you found the test data to get your results ?

**Comment by ForestsKing** [comment]
Sorry for the delayed response. Unfortunately, due to a computer replacement, I am currently unable to locate the evaluation code. However, I have found the evaluation dataset and uploaded it to [Google Drive](https://drive.google.com/drive/folders/1NQJT_Pjd1PM9d5cDiJgDBW36SddHdepu?usp=drive_link).  To reduce evaluation costs, we did not use the entire Time-Series-Library dataset. Instead, we only utilized the last dimension of the test set (as the training set was needed for training other Full-Shot Forecast models) and applied a non-overlapping sliding window approach. I hope this helps!
